packages/app-desktop/gui/NotePropertiesDialog.tsx

```diff
@@ -310,6 +312,8 @@ class NotePropertiesDialog extends React.Component<Props, State> {
 							// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
 							onKeyDown: (event: any) => onKeyDown(event),
 							style: styles.input,
+							id: key,
+							name: key,
```

Comment:
```suggestion
							id: uniqueId(key),
							name: uniqueId(key),
```
**Note**: There's also a conflict here!